static/app/components/segmentedControl.tsx

```diff
@@ -110,8 +110,8 @@ function Segment({
 
   const {inputProps} = useRadio({...props}, state, ref);
 
-  const prevOptionIsSelected = state.selectedValue === prevKey;
-  const nextOptionIsSelected = state.selectedValue === nextKey;
+  const prevOptionIsSelected = prevKey && state.selectedValue === prevKey;
+  const nextOptionIsSelected = nextKey && state.selectedValue === nextKey;
```

Comment:
Fix a hvoer overflow issue when no segment is selected:

**Before ——**
<img width="333" alt="Screenshot 2023-02-02 at 11 20 42 AM" src="https://user-images.githubusercontent.com/44172267/216429128-35ad1a54-ed67-41e6-babc-f319c2925f67.png">

**After ——**
<img width="326" alt="Screenshot 2023-02-02 at 11 19 14 AM" src="https://user-images.githubusercontent.com/44172267/216428963-8539b186-aa5e-4b20-9387-0d52bf99ebd6.png">